adagucserverEC/CRequest.cpp

```diff
@@ -1177,6 +1177,20 @@ int CRequest::fillDimValuesForDataSource(CDataSource *dataSource, CServerParams
       }
     }
 
+    // Check if requested time dimensions use valid characters
```

Comment:
This check was done in `CDBAdapterPostgreSQL.cpp`, however this is pure input validation. For now I moved it to `CRequest.cpp` since that is where other input validation already happens. Maybe it could help to move all input validation to a separate cpp file.

Reply:
Can you write an issue in github for this?

Reply:
Created https://github.com/KNMI/adaguc-server/issues/398